parser.go

```diff
@@ -49,6 +49,19 @@ const (
 
 	minVersion = 1
 	maxVersion = 9
+
+	rdbModuleOpcodeEOF    = 0
+	rdbModuleOpcodeSInt   = 1
+	rdbModuleOpcodeUInt   = 2
+	rdbModuleOpcodeFloat  = 3
+	rdbModuleOpcodeDouble = 4
+	rdbModuleOpcodeString = 5
+
+	redisBloomBloomFilter    = 3465209449566631940
```

Comment:
It would be better add a source to where these constants are from.

Reply:
I have added some comments about the constants. 
Attached here with example code to verify the constants https://cplayground.com/?p=loris-caribou-goosander